src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    // Here I'll add the reducers for categories
+  },
```

Comment:
Nice job so far👍. Just as required for this milestone, we are required to add a reducer that checks the status😉 Kindly see the image below and make the necessary change👍.

<img width="772" alt="image" src="https://github.com/VelzckC0D3/bookstore-react/assets/64643995/9d6b9abd-aa8e-4b30-84e6-e2626f9a0bd6">

Reply:
@Ibizugbe thank you! I didn't noticed that I didn't add it, corrected now !